index.js

```diff
@@ -49,6 +57,15 @@ var encodingSupported = ['*', 'gzip', 'deflate', 'identity']
 
 function compression (options) {
   var opts = options || {}
+  var optsBrotli = objectAssign({}, opts.brotli)
+
+  if (hasBrotliSupport) {
+    var brotliParams = {}
+    brotliParams[zlib.constants.BROTLI_PARAM_QUALITY] = 4
+
+    // set the default level to a reasonable value with balanced speed/ratio
+    optsBrotli.params = Object.assign(brotliParams, optsBrotli.params)
```

Comment:
This is a great idea, love it, why not just assign for optsBrotli too and remove the dep entirely?

Reply:
Oh, didn’t realize I’m using `Object.assign` directly here. I’ll remove that dependency.